Mathlib/Data/Matrix/Mul.lean

```diff
@@ -1,7 +1,8 @@
 /-
 Copyright (c) 2018 Ellen Arlt. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Ellen Arlt, Blair Shi, Sean Leather, Mario Carneiro, Johan Commelin, Lu-Ming Zhang
+Authors: Ellen Arlt, Blair Shi, Sean Leather, Mario Carneiro, Johan Commelin, Lu-Ming Zhang,
+  Matteo Cipollina
```

Comment:
I'm happy to add you here but we usually only add people to the "Authors" list when they write a lot of a file. So adding you here is fine but a bit unusual.